src/create_isurf.cpp

```diff
@@ -171,7 +171,7 @@ void CreateISurf::command(int narg, char **arg)
   
   tvalues = NULL;
   int pushflag = 0;
-  char *sgroupID = arg[0];
+  char *sgroupID = 0;
```

Comment:
Please set this to `NULL` instead of `0`.

Reply:
Made this change